.github/workflows/ci.yml

```diff
@@ -74,6 +74,11 @@ jobs:
     timeout-minutes: 45
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checkout all commits to be able to determine merge base for GIB
```

Comment:
Since we rely on `git describe`, we actually always wanted unlimited fetch depth, right? Not GIB-specific?

Reply:
There were existing jobs that had a depth of 0, with a comment, and they remain unchanged. Since this jobs only needs this for GIB, I added a specific comment explaining that.